images/builder/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+- test-infra-oncall
+approvers:
+- test-infra-oncall
```

Comment:
wg-k8s-infra-oncall should be here too

Reply:
i'm not sure on this, can we do this in a follow-up?

Reply:
sure